src/main/java/com/shima/todo_list/service/TodoListService.java

```diff
@@ -51,4 +51,11 @@ public void update(int id, LocalDate scheduledEndDate, LocalDate actualEndDate)
         TodoList todoList = new TodoList(id, scheduledEndDate, actualEndDate);
         todoListMapper.update(todoList);
     }
+
+    //DELETE(指定したid削除）
+    public void delete(int id) {
+        todoListMapper.findById(id)
+                .orElseThrow(() -> new TaskNotFoundException("Task information not found"));
```

Comment:
メッセージをTask deletedにするのであればこちらのinformationも消したい。
統一感大事。
```suggestion
                .orElseThrow(() -> new TaskNotFoundException("Task not found"));
```

Reply:
Messageを統一致しました。

d037841543b8459b2d21f683f4eac0846ca69aa3

Reply:
ok